scripts/vscode.patch

```diff
@@ -824,6 +824,15 @@ index 74fc798..03728d0 100644
 +			// 	supportFetchAPI: true,
 +			// 	corsEnabled: true
 +			// });
+@@ -310 +328 @@
+-
++			
+@@ -328 +346,4 @@
+-					newFrame.contentWindow.focus();
++					/* Comment out to avoid VS Code from focusing on markdown preview on reload */
```

Comment:
This comment doesn't match the behavior; maybe something like "Only focus the updated frame if it already had focus"?

Reply:
fixed